Dockerfile

```diff
@@ -233,7 +206,7 @@ ENV AIRFLOW_BRANCH=${AIRFLOW_BRANCH}
 ENV AIRFLOW_GITHUB_DOWNLOAD=https://raw.githubusercontent.com/${AIRFLOW_REPO}/${AIRFLOW_BRANCH}
 
 # Airflow Extras installed
-ARG AIRFLOW_EXTRAS="all"
+ARG AIRFLOW_EXTRAS="all,stastd,elasticsearch"
```

Comment:
Are statsd and elasticsearch not already included via all?

Reply:
The statsd is not, elasticsearch is.  

I am not sure we have an understanding about what different extras mean actually. When you look at setup.py currently we have: **devel_ci** = **devel_all** = **all** (except in PY3 devel_ci does not contain snakebite).

We also have:
* **devel_minreq** (devel + kubernetes + mysql + doc + password + cgroups)
* **devel_hadoop** (devel_minreq + hive + hdfs + webhdfs + kerberos and devel_hadoop).

So we might use the opportunity to clean it up !

My proposal is only to leave *devel* and *all* and make sure *all* does not contain devel. Then for CI we would have [all,devel] extras and for Prod images only [all].

We can also leave *devel_minreq* and *devel_hadoop* as they are good for those casese.

Reply:
> My proposal is only to leave _devel_ and _all_ and make sure _all_ does not contain devel. Then for CI we would have [all,devel] extras and for Prod images only [all].
> 

Sounds like a plan to me.

> We can also leave _devel_minreq_ and _devel_hadoop_ as they are good for those cases.

Also good.